.github/workflows/build.yml

```diff
@@ -1,6 +1,6 @@
 name: Build All Solvers
 
-on: 
```

Comment:
You should revert all changes to `build.yml`.

Reply:
Done.